src/main/java/org/ambraproject/wombat/controller/ArticleMetadata.java

```diff
@@ -222,6 +225,16 @@ RevisionMenu getRevisionMenu() throws IOException {
     return new RevisionMenu(revisionList, isDisplayingLatestRevision);
   }
 
+
+  String getPeerReview() throws IOException {
```

Comment:
Instead of reading XML let's call the rhino API:  http://localhost:8082/articles/10.1371%2B%2Bjournal.pone.0178719/ingestions/1/items

Look for items like this one:
```
"10.1371/journal.pone.0178719.r002": {
      "doi": "10.1371/journal.pone.0178719.r002",
      "itemType": "reviewLetter",
      "files": {
        "letter": {
          "bucketName": "corpus",
          "crepoKey": "10.1371/journal.pone.0178719.r002.xml",
          "crepoUuid": "c391cb75-b3c1-4c7a-8916-70efff5b0e43",
          "size": 23347
        }
      }
    } 
```